src/index.js

```diff
@@ -58,6 +58,13 @@ const extendCodeBlockWithSyntaxHighlighting = ( settings ) => {
 				( languageOption ) => languageOption.label.toLowerCase()
 			);
 
+			const plainTextProps = {
+				value: attributes.content,
+				onChange: ( content ) => setAttributes( { content } ),
+				placeholder: __( 'Write 2code…', 'syntax-highlighting-code-block' ),
```

Comment:
```suggestion
				placeholder: __( 'Write code…', 'syntax-highlighting-code-block' ),
```

Don't think the 2 is intentional here 😅

Reply:
Oops!